src/components/CollectionRow/CollectionRow.js

```diff
@@ -84,7 +89,7 @@ const CollectionRow = ({
   const recordNotesCount = collectionRecords.reduce((acc, { notes }) => (
     notes ? acc.concat(Object.keys(notes)) : acc), []).length;
   const savedRecordsCount = collectionRecords.filter((record) => record.saved === true).length;
-
+  const show_purpose = (collection.purpose.length) > 0
```

Comment:
No snake_case, just camelCase > `showPurpose`. This should apply throughout.

Reply:
resolved